src/model/grid-config.ts

```diff
@@ -28,13 +33,14 @@ export class GridConfig implements IGridConfig {
     /**
      * Data of grid config
      */
-    data?: any[];
+    data?: T[];
 
     /**
      * Creates an instance of grid config.
      */
     constructor(){
         this.renderCompleteSubscription = new BehaviorSubject(false);
+        this.rowHeight = 40;

```

Comment:
```suggestion
        this.rowHeight = 40;
```
Need a better way to define 40, seems like a hardcoded value.

Reply:
Perhaps from a static class with constants (default values)